pages/home.jsx

```diff
@@ -295,11 +303,13 @@ var HomePage = React.createClass({
               linkTo="activities"
               imgSrc="/img/pages/home/svg/icon-teachanactivity.svg"
               head="Teach an Activity"
+              onClick={this.handleTeachBtnClick}
             />
             <IconButton
               linkTo="mozilla-clubs"
               imgSrc="/img/pages/home/svg/icon-startamozillaclub.svg"
               head="Start A Mozilla Club"
+              onClick={this.handleClubBtnClick}
```

Comment:
I just want to double check that these `IconButton`s still redirect to the `linkTo` address after adding this `onClick` event. I expect they do, but I'm not setup to run the code locally right now. 

Other than that, the ga tracking code looks great to me :+1: